ssg/build_derivatives.py

```diff
@@ -44,14 +44,18 @@ def add_cpes(elem, namespace, mapping):
     return affected
 
 
-def add_cpe_item_to_dictionary(tree_root, product_yaml_path, cpe_ref, id_name):
+def add_cpe_item_to_dictionary(tree_root, product_yaml_path, cpe_ref, id_name, cpe_items_dir):
     cpe_list = tree_root.find(".//{%s}cpe-list" % (PREFIX_TO_NS["cpe-dict"]))
     if cpe_list:
         product_yaml = load_product_yaml(product_yaml_path)
-        product_cpes = ProductCPEs(product_yaml)
+        product_cpes = ProductCPEs()
+        product_cpes.load_cpes_from_directory_tree(cpe_items_dir, product_yaml)
         cpe_item = product_cpes.get_cpe(cpe_ref)
         translator = IDTranslator(id_name)
-        cpe_item.check_id = translator.generate_id("{" + oval_namespace + "}definition", cpe_item.check_id)
+        cpe_item.conditional[
+                             "oval_id"] = translator.generate_id(
+                             "{" + oval_namespace + "}definition",
+                             cpe_item.conditional["oval_id"])
```

Comment:
The coding style here isn't great.

Reply:
Indeed, please use explanatory variables.

Reply:
also indentation

Reply:
I tried to fix the string in this commit: https://github.com/ComplianceAsCode/content/pull/8923/commits/e3f882e7997ee314941e298779f99ba89951054a
Indentation is fixed as well.
Not sure how to fix variable names... I think they are quite clear.